<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestaoPedido</class>
 <widget class="QDialog" name="GestaoPedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEST√ÉO DE PEDIDO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/img/img/pedido.png</normaloff>:/img/img/pedido.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBoxGestaoPedido">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>681</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>228</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutGPListar">
     <item>
      <layout class="QVBoxLayout" name="layoutGPID">
       <item>
        <widget class="QLabel" name="labelGPID">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGPID"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGPListar">
         <property name="text">
          <string>Listar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerGPB">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutGPCategoria">
       <item>
        <widget class="QRadioButton" name="radioButtonGPCliente">
         <property name="text">
          <string>Cliente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonGPPedido">
         <property name="text">
          <string>Pedido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonGPTodos">
         <property name="text">
          <string>Todos</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetGPLista">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>921</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>250</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>250</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>940</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>940</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>3</width>
      <height>100</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>21</y>
      <width>3</width>
      <height>100</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>3</width>
      <height>520</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>150</y>
      <width>3</width>
      <height>520</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Recursos.qrc">:/img/img/pedido.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>15</y>
     <width>1031</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Recursos.qrc">:/img/img/pucpb.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditGPID</tabstop>
  <tabstop>pushButtonGPListar</tabstop>
  <tabstop>radioButtonGPCliente</tabstop>
  <tabstop>radioButtonGPPedido</tabstop>
  <tabstop>radioButtonGPTodos</tabstop>
  <tabstop>listWidgetGPLista</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
